lib/parser/ccda/sections/social_history.js

```diff
@@ -43,6 +43,9 @@ var exportSocialHistorySection = function (version) {
         }
         delete this.js.nullCodeReplacementText;
       }
+      if (!_.has(this, "js.identifiers")) {
+        this.js.identifiers = [{"identifier": null}];
```

Comment:
Hey I'm just curious because I don't really understand what I'm reading here
https://github.com/search?q=repo%3Aelationemr%2Fblue-button%20identifiers&type=code
It looks like "identifiers" is defined as "0..*" field in this file (line 23). Should it be changed as well ?

Reply:
yeah, when the xml has nullFlavor it'll get ignored `<id nullFlavor="UNK"/>`, it won't get parsed. This code is in another blue-button-xml repo.